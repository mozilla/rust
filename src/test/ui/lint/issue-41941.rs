// Checks whether exceptions can be added to #![deny(warnings)].
// check-pass

#![deny(warnings)]
#![warn(unused_variables)]

fn main() {
    let a = 5;
    //~^ WARNING: unused
}
