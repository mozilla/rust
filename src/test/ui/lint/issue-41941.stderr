warning: unused variable: `a`
  --> $DIR/issue-41941.rs:8:9
   |
LL |     let a = 5;
   |         ^ help: if this is intentional, prefix it with an underscore: `_a`
   |
note: the lint level is defined here
  --> $DIR/issue-41941.rs:5:9
   |
LL | #![warn(unused_variables)]
   |         ^^^^^^^^^^^^^^^^

warning: 1 warning emitted

